README.md

```diff
@@ -352,7 +352,9 @@ Here is an example with `ngx_lua`:
 
 ```nginx
   # Nginx, configuration
-
+  # In the recent OpenResty, you need to include path of cjson.so using
+  # lua_package_cpath "/path/to/openresty/cjson.so";
+  # lua_load_resty_core off; #uncomment if lua_load_resty_core is buggy

```

Comment:
Is there an issue about that? I feel it a bit impolite to say that something is buggy w/o any link.

Reply:
It looks like one https://github.com/openresty/lua-nginx-module/issues/1509